components/pagination/metadata/mods.md

```diff
@@ -6,5 +6,3 @@
 | `--mod-pagination-counter-margin-inline-start` |
 | `--mod-pagination-page-button-inline-spacing`  |
 | `--mod-pagination-textfield-width`             |
-| `--mod-textfield-min-width`                    |
```

Comment:
Did you mean for this to go in as well? If so, could you separate out this change to its own commit? 😬

Reply:
I can pull it out - it's a side-effect of the updated mods script going in. Probably pagination's PR was merged after the mods script was opened for code review.